src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -769,7 +769,7 @@ public class C
     public delegate (int, int) F();
 }";
             var source1 = @"
-public struct C
```

Comment:
We should compare class vs. class here not class vs. struct. The change happens in the delegate not in the class/struct.

---
In reply to: [116844195](https://github.com/dotnet/roslyn/pull/19283#discussion_r116844195) [](ancestors = 116844195)